canisters/node_status_canister/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[workspace]
+
+members = [
+    "src/node_status_canister_backend",
+]
+
+resolver = "2"
```

Comment:
If we anticipate to have more canisters (which i think we do) shouldn't we put a workspace file in `canisters/Cargo.toml`?

Reply:
Good point, let me move it